### ISSUE TITLE
Question about data used for training

### ISSUE THREAD

**Issue by maybeliuchuan** [cardinalblue/clip-models-for-distillation#3]
Could you please share the data used for training? 
In the related paper, you methioned that "We began with a dataset of ~200,000 images taken from various sources. After about 10 epochs, once we began to see some promising results, the size was increased to 800,000+ images."
Besides, could you share the whole code of this project, including the dataloader and the loss function design? Thank you very much.
